sdk/loadtestservice/Azure.Developer.LoadTesting/samples/Sample1_CreateOrUpdateTestAsync.md

```diff
@@ -2,21 +2,22 @@
 
 To use these samples, you'll first need to set up resources. See [getting started](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/loadtestservice/Azure.Developer.LoadTesting/README.md#getting-started) for details.
 
-You can create a LoadTestclient and call the `CreateOrUpdateTest` method from client `LoadTestAdministrationClient`
+The sample below demonstrates how to create a new load test using the `LoadTestAdministrationClient` client.
 
 ## Create LoadTestAdministrationClient
 ```C# Snippet:Azure_Developer_LoadTesting_CreateAdminClient
-string endpoint = TestEnvironment.Endpoint;
-Uri enpointUrl = new Uri("https://"+endpoint);
-TokenCredential credential = TestEnvironment.Credential;
+// The data-plane endpoint is obtained from Control Plane APIs with "https://"
```

Comment:
How is it obtained? Let's include an example, especially if there is an az CLI command or something that can fetch it easily.

Reply:
Same with all other occurrences of this comment.

Reply:
Should we add this to Sample, https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/loadtestservice/Azure.Developer.LoadTesting#data-plane-endpoint?